lib/components/PolicyMap/PolicyMap.js

```diff
@@ -0,0 +1,223 @@
+/* eslint-disable no-magic-numbers */
+
+import React, { useState } from 'react';
+import ReactTooltip from 'react-tooltip';
+import PropTypes from 'prop-types';
+import cn from 'classnames';
+import { ComposableMap, Geographies, Geography, ZoomableGroup } from 'react-simple-maps';
+import numeral from 'numeral';
+
+import { vds } from '../../styles/style_dict/tokens/color/base_colors.json';
+import styles from './PolicyMap.css';
+import geoData from './geoData110m.json';
+
+const COLOR_SCALE = [
+  vds.color.blue.lightest.value,
+  vds.color.blue.lighter.value,
+  vds.color.blue.primary.value,
+  vds.color.blue.dark.value,
+];
+const BORDER_COLOR = vds.color.slate.darker.value;
+const WHITE = vds.color.white.value;
+const INACTIVE_COLOR = vds.color.slate.evenLighter.value;
+
+/**
+ * Renders the map component
+ * @param {Array<Object>} data An array of data in the format [{ id, value }] where ID is the 2-character ISO country code
+ * e.g. [{ id: 'US', value: 5708 }, { id: 'FR', value: 376 }]
+ * @param {Function} onClick A callback function that is used when a country is clicked. Passes { id, value }
+ * e.g. { id: 'US', value: 5708 }
+ * @param {Number} focusQuartile The quartile of data to focus on - accepts 0 to 4 where 0 is show all.
+ */
+const PolicyMap = ({ data, onClick, focusQuartile, onFocusQuartile, labelText }) => {
+  const toggleQuartile = (quartileClicked) => {
+    onFocusQuartile(focusQuartile === quartileClicked ? 0 : quartileClicked);
+  };
+  const [tooltip, setTooltip] = useState();
+
+  let sorted = data.sort((a, b) => {
+    if (a.value < b.value) return -1;
+    if (a.value > b.value) return 1;
+    return 0;
+  });
+
+  // Breakpoints used for legend
+  const breakpoints = [
+    numeral(sorted[Math.floor(0.25 * sorted.length)].value).format('0a'),
+    numeral(sorted[Math.floor(0.25 * sorted.length)].value + 1).format('0a'),
+    numeral(sorted[Math.floor(0.5 * sorted.length)].value).format('0a'),
+    numeral(sorted[Math.floor(0.5 * sorted.length)].value + 1).format('0a'),
+    numeral(sorted[Math.floor(0.75 * sorted.length)].value).format('0a'),
+    numeral(sorted[Math.floor(0.75 * sorted.length)].value + 1).format('0a'),
+    numeral(sorted[sorted.length - 1].value).format('0a'),
+  ];
+
+  sorted = sorted.map((dataPoint, i) => {
+    const colorIndex = Math.floor((i / sorted.length) * COLOR_SCALE.length);
+
+    // Color the country according to quartile if none is selected
+    if (!focusQuartile) {
+      return { ...dataPoint, color: COLOR_SCALE[colorIndex] };
+    }
+
+    // Is a quartile is selected, only color the country if it falls in the quartile range
+    if (focusQuartile === colorIndex + 1) {
+      return { ...dataPoint, color: COLOR_SCALE[colorIndex] };
+    }
+    return {};
```

Comment:
is it intentional? tooltip does not show on inactive countries when we return empty object here. if we do want to always show a tooltip, we should return `defaultPoint`

Reply:
yeah the goal was to completely disable countries outside of active quartile